data/fields/hazard_highway.json

```
@@ -0,0 +1,31 @@
+{
+    "key": "hazard",
+    "type": "combo",
+    "strings": {
+        "options": {
+            "animal_crossing": "Animal Crossing",
+            "bump": "Bump",
+            "children": "Children",
+            "curve": "Curve",
+            "curves": "Multiple Curves",
+            "cyclists": "Cyclists",
+            "dangerous_junction": "Dangerous Junction",
```

https://wiki.openstreetmap.org/wiki/Tag:hazard=dangerous_junction should be fixed first, right now it looks like permitting 100% subjective tagging

while https://wiki.openstreetmap.org/wiki/Key:hazard has ***signed*** hazards on roadways
```suggestion
            "dangerous_junction": "Junction Signed As Dangerous",
```

---

This point may be able to be resolved by changing label of 'hazard' tag as the suggestion below. It clarifies that all values under this tag are signed.